lib/livebook/application.ex

```diff
@@ -121,7 +121,8 @@ defmodule Livebook.Application do
     end
   end
 
-  defp access_url() do
+  @doc false
```

Comment:
@jonatanklosko: do we want to `@doc false` this or make it public here or make it public in another module?

Reply:
Ah the module is already doc false, so I'd make it public. Perhaps it could be in `LivebookWeb.Endpoint`, but either place seems fine to me.